cli/installer/choco/azd.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>azd</id>
+    <version>$VERSION$</version>
```

Comment:
Just confirming this is a semver1 value? Chocolatey does not currently support SemVer2.

Reply:
Our plan is to release prerelease packages (`0.x.y`) until we reach version `1.0.0` and then to only release the GA packages on package managers going forward (i.e. no prerelease/beta packages). 

Working with @heaths we came up with a [version scheme](https://github.com/Azure/azure-dev/blob/main/eng/scripts/Get-MsiVersion.Tests.ps1) which matches MSI versioning (a more narrow scheme than Semver) so that the packages will upgrade as expected.